index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0" />
+    <title>NowYou!</title>
+
+    <!-- CSS  -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="css/materialize.css" type="text/css" rel="stylesheet" media="screen,projection" />
+    <link href="css/style.css" type="text/css" rel="stylesheet" media="screen,projection" />
+
+    <!-- google fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Fredoka+One" rel="stylesheet">
+
+</head>
+
+<body>
+    <nav class="light-blue lighten-1" role="navigation">
+        <!-- site title -->
+        <div class="nav-wrapper container"><a id="logo-container" href="#" class="brand-logo">NowYou!</a>
+            <!-- link to login page -->
+
+
+            <div class="row right" class="right hide-on-med-and-down">
+                <a href="SigninAuth.html" id="download-button" class="btn-large waves-effect waves-light orange">User
+                    Login</a>
+            </div>
+            <ul id="nav-mobile" class="right hide-on-med-and-down">
+                <li><a href="sass.html">Exercises</a></li>
+                <li><a href="badges.html">FAQ</a></li>
+                <li><a href="collapsible.html">The Team</a></li>
```

Comment:
sass.html, badges.html, collapsible.html don't exist, could we please put a "#" without quotes here?